package.json

```diff
@@ -49,6 +49,7 @@
     "less": "^2.5.1",
     "less-loader": "^4.0.3",
     "lodash": "^4.17.4",
+    "metrohash": "^2.3.0",
```

Comment:
Could you use a non-native dependency for testing? It doesn't have to be super fast as it's just for the test.

I want to avoid having native deps even in webpack development.

Reply:
Sure thing, i will use xxhashjs here in that case

Reply:
perfect :+1: